### PR TITLE
rephrase tab completion to close #16

### LINKED ISSUE #16: Guide 13 Tab Completion confuses people
The `cd` + `l` + `tab` leads people to actually type `cd l tab` and be confused about it 😃  Maybe rephrase for `press the tab key?`

### PR DESCRIPTION
**Pull request by gedankenstuecke**

### CODE REVIEW

**Comment by blahah** [comment]
Much better, thanks so much!

**Comment by gedankenstuecke** [comment]
Sure, happy to help. Wasn't sure how/whether it needs to be rendered before the PR, so maybe you'll still have to do it.

**Comment by blahah** [comment]
yeah I need to regenerate the HTML for recent updates
